README.md

```diff
@@ -157,6 +158,11 @@ A few caveats and callouts:
 - Blocks have access to [GitHub Primer CSS styles](https://primer.style/css/)
 - You can use both third-party _and_ relative imports in your Block code! Simply put, feel free to install any dependencies from NPM, or import a local JS/CSS file and it should be included in the final bundle.
 - Your Block entry file **must have the Block component as its default export**. If it does not, bad things will happen.
+- To make authenticated requests to the GitHub API, create a [personal access token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) and pass it when you start the dev server:
```

Comment:
might be worth adding that it should have `repo` scope to match the prototype

Reply:
might also be worth supporting `.env` files (probably need the `dotenv` lib)

Reply:
Vite is ON IT https://vitejs.dev/guide/env-and-mode.html#env-files